### DISCUSSION TITLE
Adding background image in quarto by using "file path"  in R does not work.

### DISCUSSION THREAD

**Discussion by elcansaito** [quarto-dev/quarto-cli#8204] [Q&A]
### Description

---
title: "Evaluation of Selection Strategies"
subtitle: "Me"
author: Nobody
date: "`r Sys.Date()`"
date-format: "YYYY-MMM-D"
format: 
  revealjs:
    chalkboard: true
    slide-number: true
    width: 1600
    height: 900
    css: ["theme.css"]
    theme: simple
  pdf: default
editor: source
execute: 
  echo: true
editor_options: 
  chunk_output_type: inline
  
---

```{r echo=FALSE}
imagePath <- file.path(Sys.getenv("HOME"), "Dropbox/Thesis/Presentation/2023-2024/images/Differentcrops.png")
```

# `r fontawesome::fa("bullseye", "black")` Learn maths {data-background-image= `r imagePath` .black-text-slide}

I can´t see the background of the image using this code. Does anyone have a solution?  Below is what I got in the slide of the presentation rather that the image background

 Learn maths {data-background-image= /Users/jis_casa/Dropbox/Thesis/Presentation/2023-2024/images/Differentcrops.png .black-text-slide}

However, If I do the following (below)  it works!!. 

# `r fontawesome::fa("bullseye", "black")` Learn maths {data-background-image="/Users/jis_casa/Library/CloudStorage/Dropbox/Thesis/Presentation/2023-2024/images/Differentcrops.png" .black-text-slide}

If I add "Library/CloudStorage" in the path does not work either 

"imagePath <- file.path(Sys.getenv("HOME"), "/Library/CloudStorage/Dropbox/AA_UPM/CLASSES/2023-2024/3_CIHEAM/Quarto_Lecture/images/Differentcrops.png")"

**Comment by mcanouil** [comment]
Could you share and format a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Reply by cderv** [reply]
@elcansaito especially can you at least format your issue correctly ? Guideline at: https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us
